reaper.go

```diff
@@ -74,6 +74,11 @@ func NewReaper(ctx context.Context, sessionID string, provider ReaperProvider, r
 		WaitingFor: wait.ForListeningPort(listeningPort),
 	}
 
+	tcConfig := readTCPropsFile()
```

Comment:
I think this will work, although I'd like to read the properties just once, when we create the Docker Provider.

IIRC the Provider is obtained when the container is created. Could you double check it please?

This is not a blocker, so we can merge this as is if we do not find a better path.

Reply:
Yes, that is true. I've added the `Config() TestContainersConfig` to the `DockerProvider` to fetch the configuration that is now read when `NewDockerProvider` is invoked.